src/Engine/Util/Geometry.hpp

```diff
@@ -114,15 +114,16 @@ struct Intersect {
     glm::vec3 norm; // the normal of the surface of the object at the point of intersection
     bool face; // true if the ray hit the outside surface, false if inside surface
 
-    Intersect::Intersect() :
+    Intersect() :
         is(false),
         dist(std::numeric_limits<float>::infinity()),
         loc(),
         norm(),
         face(true)
     {}
 
-    Intersect::Intersect(
+    Intersect(
+        bool is,
```

Comment:
why did you add this field in? Would you ever construct an intersect with parameters that did not intersect?

Reply:
I didn't add any fields

Reply:
Should this constructor be different?